src/it/resources/org/checkstyle/suppressionxpathfilter/staticvariablename/SuppressionXpathRegressionStaticVariableName2.java

```diff
@@ -0,0 +1,7 @@
+package org.checkstyle.suppressionxpathfilter.staticvariablename;
+
+public class SuppressionXpathRegressionStaticVariableName2 {
+
+     public int num1; // OK
+     protected static int NUM2; // warn
```

Comment:
It is almost same as input1, please wrap this field in inners class to show violation on field of inner class.